backup/crowbar-backup

```diff
@@ -246,6 +248,13 @@ function restore()
   fi
   echo
 
+  # Make sure install finished correctly
+  if ! [ -e /opt/dell/crowbar_framework/.crowbar-installed-ok ]; then
+    echo "Crowbar ".crowbar-install-ok" marker missing"
```

Comment:
Interesting, this was also broken in qa_crowbarsetup.sh. Fixed there now as well.